### PR TITLE
Helya: Mythic Additions

### PR DESCRIPTION
**Pull request by Justw8**
See Method's video (once out) to confirm most timers.

### CODE REVIEW

File: TrialOfValor/Helya.lua [hunk 1]
```diff
@@ -155,13 +181,16 @@ end
...
-	self:Bar(228054, 19.5) -- Taint of the Sea
-	self:Bar(229119, 31, CL.count:format(self:SpellName(229119), orbCount)) -- Orb of Corruption
-	self:Bar(228730, 37) -- Tentacle Strike
+	tentacleCounter = 1
+	
+	self:Bar(227967, self:Mythic() and 10.5 or 12) -- Bilewater Breath
+	self:Bar(228054, self:Mythic() and 15.5 or 19.5, CL.count:format(self:SpellName(228054), taintCount)) -- Taint of the Sea
+	self:Bar("orb_ranged", self:Mythic() and 14 or 31, CL.count:format(L.orb_bar:format("Ranged"), orbCounter), 229119) -- Orb of Corruption
+	self:Bar(228730, self:Mythic() and 35.3 or 36.7, CL.count:format(self:SpellName(228730), tentacleCounter)) -- Tentacle Strike
+	self:Mythic() and self:Berserk(660)
```
Comment (funkydude) [inline TrialOfValor/Helya.lua]: This isn't valid syntax, which is why the build is failing. Should be an if statement.

**Comment by Justw8** [comment]
updated.

**Comment by funkydude** [comment]
Are you planning on cleaning this up further or are you happy with this being merged?

File: TrialOfValor/Helya.lua [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
--- - Fix/Remove untested mythic funcs:
---   - MistInfusion
+-- Fix/Remove untested mythic funcs:
+-- MistInfusion
+-- Update Tentacle Strike to use table (rough edit atm)
+-- Update Orb of Corrosion timers p3 into table (rough edit atm)
+-- Localize Orb bars better?
+-- (Mythic) Update Lantarn of Darkness initial timer.
+-- (Mythic) Update Fetid Rot timers 
+-- (Mytchi) If marking Orb targets, in p3 there is double dps
```
Comment (funkydude) [inline TrialOfValor/Helya.lua]: Minor typo

**Comment by Justw8** [comment]
Last cleanup done now, rest is up to you guys :) Might missing timers when I get new transcripts next reset and they aren't here yet. (they weren't important for us, so didn't update/check them and lost transcripts due to logs being too big :(. )

**Pull request merged**